### ISSUE TITLE
feat(web): Enable ZE and ZFTP to be used on `VSCode Web`

### ISSUE THREAD

**Issue by zFernand0** [zowe/vscode-extension-for-zowe#1798]
On October 20th, 2021, Microsoft [blogged](https://code.visualstudio.com/blogs/2021/10/20/vscode-dev) about [VSCoode Web](https://code.visualstudio.com/docs/editor/vscode-web).

It seems that most Zowe extensions are still not enabled.
![image](https://user-images.githubusercontent.com/37381190/167845497-8de23ef1-1f22-441a-aa69-a954b57d4d62.png)
👀 The enhancement request is [to enable](https://code.visualstudio.com/docs/editor/vscode-web#_extensions) Zowe Explorer (and others) to be used on `VSCode Web`. 👀 

---

Imagine the following interaction: (#1782)
- `Zowe Bot`: Hello `<user>`, your `JOB01234` ended with `ABEND S222`. Would you like to see more details ?
- `<user>`&nbsp;&nbsp;&nbsp;: Yes please
- `Zowe Bot`: Alright! [`https://github.dev/<user>/__zowe-bot__debug-job01234`](https://github.dev/zowe/vscode-extension-for-zowe)

The URL will:
- Redirect you to a repository with Zowe Explorer already installed 
- Your connection details specified in the `zowe.config.json` file
- The `JOB01234` already opened in the Job Tree
- The `DD=JESMSGLG` already open in the editor
---
Other extensions:
![image](https://user-images.githubusercontent.com/37381190/167845404-1db0fdf2-93cc-4742-84e2-36055f34d118.png)

**Comment by traeok** [comment]
Looks like we can now download Zowe Explorer within VSCode web. However, the core functionality seems to be broken. It looks similar to what was going on in #1854 , but now it seems that it is functional in Codespaces and not in VSCode Web 🤔

**Comment by JillieBeanSim** [comment]
@zFernand0 I agree that shouldn't support this for now.  @anaxceron we can document this if documenting environments we don't support, if we only documenting what we do support we can just close this for now as not planned.
